adding 3 files

narismadz opened the pull request:
- Modify column "IsIncentiveLive" column from "Boolean" to "Choice" type in Create-Incentive-SharePointList.ps1
- Comment creation of Title and Attachment column in Create-Incentive-SharePointList.ps1
- Add PowerShell script for MFA-tenant name "MFA-tenant-Create-Incentive-SharePointList.ps1"
- fixed dependency in PowerApps on Gallery to accept Choice value
- Replace toggle switch with dropdown since IsIncentiveLive is not boolean anymore

Ixinho commented:
> * Modify column "IsIncentiveLive" column from "yes/no" to "Choice" type in Create-Incentive-SharePointList.ps1
> * Comment on line Title and Attachment column in all SPO lists in Create-Incentive-SharePointList.ps1
> * Add PowerShell script for MFA-tenant name "MFA-tenant-Create-Incentive-SharePointList.ps1"
> * fixed dependency in PowerApps on Gallery to accept Choice value to fix PowerApps confusion and delegation on yes/no
> * Replace toggle switch with dropdown since IsIncentiveLive is not boolean anymore

How did you fix dependency in Powerapps? I changed the type of column in sp list itself, but what do i need to do in PowerApps in order to make that switch "Is live" work properly?

narismadz commented:
> > * Modify column "IsIncentiveLive" column from "yes/no" to "Choice" type in Create-Incentive-SharePointList.ps1
> > * Comment on line Title and Attachment column in all SPO lists in Create-Incentive-SharePointList.ps1
> > * Add PowerShell script for MFA-tenant name "MFA-tenant-Create-Incentive-SharePointList.ps1"
> > * fixed dependency in PowerApps on Gallery to accept Choice value to fix PowerApps confusion and delegation on yes/no
> > * Replace toggle switch with dropdown since IsIncentiveLive is not boolean anymore
> 
> How did you fix dependency in Powerapps? I changed the type of column in sp list itself, but what do i need to do in PowerApps in order to make that switch "Is live" work properly?

I changed SP List column "IsIncentiveLive" to choice type but have only 2 option (true, false)
Then I fixed the code in PowerApps change Is active toggle switch to be dropdown based on IsIncentiveLive's choice
![image](https://user-images.githubusercontent.com/16915716/100049429-dd95bb80-2e49-11eb-8bfa-59c0a299010e.png)


Like "Items" property of dropdown

add activity
![image](https://user-images.githubusercontent.com/16915716/100049380-c656ce00-2e49-11eb-9af7-773c29bff61b.png)


edit activity
![image](https://user-images.githubusercontent.com/16915716/100049264-7e37ab80-2e49-11eb-98df-688c7360665d.png)


Then modify gallery's display image
![image](https://user-images.githubusercontent.com/16915716/100049200-59dbcf00-2e49-11eb-9a9e-158747beb864.png)
![image](https://user-images.githubusercontent.com/16915716/100049331-a6270f00-2e49-11eb-8ddc-79a00668101e.png)

Ixinho commented:
> > > * Modify column "IsIncentiveLive" column from "yes/no" to "Choice" type in Create-Incentive-SharePointList.ps1
> > > * Comment on line Title and Attachment column in all SPO lists in Create-Incentive-SharePointList.ps1
> > > * Add PowerShell script for MFA-tenant name "MFA-tenant-Create-Incentive-SharePointList.ps1"
> > > * fixed dependency in PowerApps on Gallery to accept Choice value to fix PowerApps confusion and delegation on yes/no
> > > * Replace toggle switch with dropdown since IsIncentiveLive is not boolean anymore
> > 
> > 
> > How did you fix dependency in Powerapps? I changed the type of column in sp list itself, but what do i need to do in PowerApps in order to make that switch "Is live" work properly?
> 
> I changed SP List column "IsIncentiveLive" to choice type but have only 2 option (true, false)
> Then I fixed the code in PowerApps change Is active toggle switch to be dropdown based on IsIncentiveLive's choice
> ![image](https://user-images.githubusercontent.com/16915716/100049429-dd95bb80-2e49-11eb-8bfa-59c0a299010e.png)
> 
> Like "Items" property of dropdown
> 
> add activity
> ![image](https://user-images.githubusercontent.com/16915716/100049380-c656ce00-2e49-11eb-9af7-773c29bff61b.png)
> 
> edit activity
> ![image](https://user-images.githubusercontent.com/16915716/100049264-7e37ab80-2e49-11eb-98df-688c7360665d.png)
> 
> Then modify gallery's display image
> ![image](https://user-images.githubusercontent.com/16915716/100049200-59dbcf00-2e49-11eb-9a9e-158747beb864.png)
> ![image](https://user-images.githubusercontent.com/16915716/100049331-a6270f00-2e49-11eb-8ddc-79a00668101e.png)

Amazing, gonna test it out as soon i as i do my daily work :) Thank you a lot.
Since i see you are very very experienced with this app, would you mind answering another question of mine? Do you think it's possible to show a picture of an activity when you redeem points for the activity? Currently there is a pop up with a picture of medal along with number of points (congrats you completed blabla), but since we have cool activity pictures, i thought it would be great to show the activity picture (the one attached when creating a new activity). What  do you think?

Ixinho commented:
Also another very quick question related to the original question.
In edit mode of this app, in **Tree View**, i can navigate between User Screen and Admin Screen, but on Admin Screen i see only the main dashboard, how do i actually preview "Update Activities" tab in edit mode?

narismadz commented:
> Also another very quick question related to the original question.
> In edit mode of this app, in **Tree View**, i can navigate between User Screen and Admin Screen, but on Admin Screen i see only the main dashboard, how do i actually preview "Update Activities" tab in edit mode?

Just press play and do actual click edit or add activity and when it pops out stop play mode then you can edit an object

Ixinho commented:
> > Also another very quick question related to the original question.
> > In edit mode of this app, in **Tree View**, i can navigate between User Screen and Admin Screen, but on Admin Screen i see only the main dashboard, how do i actually preview "Update Activities" tab in edit mode?
> 
> Just press play and do actual click edit or add activity and when it pops out stop play mode then you can edit an object
> 
> For replacing the pic instead of medal on "congrats" pop out box i think it possible to do. Maybe need to set some source of variables for picture reference

Let me know if you re willing to give it a try and do it , and we can talk about your rate :) Cheers

Ixinho commented:
> > Also another very quick question related to the original question.
> > In edit mode of this app, in **Tree View**, i can navigate between User Screen and Admin Screen, but on Admin Screen i see only the main dashboard, how do i actually preview "Update Activities" tab in edit mode?
> 
> Just press play and do actual click edit or add activity and when it pops out stop play mode then you can edit an object
> 
> For replacing the pic instead of medal on "congrats" pop out box i think it possible to do. Maybe need to set some source of variables for picture reference

Managed to do it on my own, thanks for the heads up :)
Last question. I see a lot of people complained that the app stops working once there are 500 items in the lists, and some people suggested to increase the dataset from 500 to 2000. Could you please help me and tell me where and how do i increase it? We should start using this app in few days in our team, prepared all activities and awards, but 500 items will be created very fast, so i need to increase it :) Thank you in advance

narismadz commented:
> > > Also another very quick question related to the original question.
> > > In edit mode of this app, in **Tree View**, i can navigate between User Screen and Admin Screen, but on Admin Screen i see only the main dashboard, how do i actually preview "Update Activities" tab in edit mode?
> > 
> > 
> > Just press play and do actual click edit or add activity and when it pops out stop play mode then you can edit an object
> > For replacing the pic instead of medal on "congrats" pop out box i think it possible to do. Maybe need to set some source of variables for picture reference
> 
> Managed to do it on my own, thanks for the heads up :)
> Last question. I see a lot of people complained that the app stops working once there are 500 items in the lists, and some people suggested to increase the dataset from 500 to 2000. Could you please help me and tell me where and how do i increase it? We should start using this app in few days in our team, prepared all activities and awards, but 500 items will be created very fast, so i need to increase it :) Thank you in advance

This one I think

![image](https://user-images.githubusercontent.com/16915716/101507502-2fd9ef00-39a9-11eb-96d0-7cf3e27d40ae.png)

Ixinho commented:
> > > > Also another very quick question related to the original question.
> > > > In edit mode of this app, in **Tree View**, i can navigate between User Screen and Admin Screen, but on Admin Screen i see only the main dashboard, how do i actually preview "Update Activities" tab in edit mode?
> > > 
> > > 
> > > Just press play and do actual click edit or add activity and when it pops out stop play mode then you can edit an object
> > > For replacing the pic instead of medal on "congrats" pop out box i think it possible to do. Maybe need to set some source of variables for picture reference
> > 
> > 
> > Managed to do it on my own, thanks for the heads up :)
> > Last question. I see a lot of people complained that the app stops working once there are 500 items in the lists, and some people suggested to increase the dataset from 500 to 2000. Could you please help me and tell me where and how do i increase it? We should start using this app in few days in our team, prepared all activities and awards, but 500 items will be created very fast, so i need to increase it :) Thank you in advance
> 
> This one I think
> 
> ![image](https://user-images.githubusercontent.com/16915716/101507502-2fd9ef00-39a9-11eb-96d0-7cf3e27d40ae.png)

Perfect, thank you.
